Tools/CISupport/ews-build/steps.py

```diff
@@ -120,10 +126,11 @@ def commit_status_url(cls, sha, repository_url=None):
         return '{}/statuses/{}'.format(api_url, sha)
 
     @classmethod
-    def credentials(cls):
+    def credentials(cls, user=None):
         try:
             passwords = json.load(open('passwords.json'))
-            return passwords.get('GITHUB_COM_USERNAME', None), passwords.get('GITHUB_COM_ACCESS_TOKEN', None)
+            prefix = f"GITHUB_COM_{user.upper().replace('-', '_')}" if user else 'GITHUB_COM_'
+            return passwords.get(f'{prefix}USERNAME', None), passwords.get(f'{prefix}ACCESS_TOKEN', None)
```

Comment:
adding an entry in passwords.json for MERGE_QUEUE_USERNAME and MERGE_QUEUE_PASSWORD might be confusing/misleading as the name wouldn't indicate anything about GITHUB. We should have GITHUB in these keys.

Also would be good if we can fall back to standard key (e.g.: GITHUB_COM_ACCESS_TOKEN) in case queue specific key is not found.

Reply:
It would be `GITHUB_COM_MERGE_QUEUE_USERNAME` and `GITHUB_COM_MERGE_QUEUE_ACCESS_TOKEN` with the current code